### ISSUE TITLE
[Bug]: Non-tiny presets still train the image generator at 512 and are OOM-killed

### ISSUE THREAD

**Issue by mantreshkhurana** [Spyxpo/framerai#166]
### Preflight checklist

- [x] I searched existing issues and did not find a duplicate.
- [x] I am running the latest version of the affected branch.

### Affected component

Training / data pipeline

### Description

Follow-up to #121, which fixed the tiny presets only.

`image_train_resolution` defaults to 512 in `model/configs/model_config.py` and is
overridden only by `framer-tiny` and `framer-tiny-moe` (both to 64). Every other
preset, including the single-GPU targets `framer-small`, `framer-medium`,
`framer-large`, `framer-3b` and `framer-8b`, still trains the image generator at
512x512.

Memory on that path grows roughly with the square of the resolution. The
measurements in #121 put one forward and backward at batch size 1 at 3.0 GB at
64, 37 GB at 128, and killed at 256 and above, and `build.py` uses
`batch_size // 2` rather than 1. `--train-modalities` on any non-tiny preset is
therefore not runnable on the hardware those presets are documented for.

### Steps to reproduce

1. Prepare the example data as documented in data/README.md.
2. Run `python build.py --mode all --size small --data-dir data/examples --train-modalities`.
3. Text training completes, then the image generator stage is OOM-killed.

### Expected behavior

Every preset should carry an `image_train_resolution` it can actually train at on
the hardware that preset targets, or the image generator path should use
gradient checkpointing so the resolution is not the binding constraint. A run
that cannot fit should fail with a clear message naming the setting rather than
being killed by the OOM killer.

### Logs and error output

```shell
Killed
```

### Environment

Model (Python 3.10+). Reproduced on CPU with 68 GB RAM in #121; the same scaling
applies on GPU against VRAM.

**Comment by kunaldeo001** [comment]
I want to take #166 next.

**Comment by mantreshkhurana** [comment]
Fixed in #185, now on stable via 3d15b91. Presets carry an `image_train_resolution` they can actually train at, and `validate()` rejects a pixel U-Net above 64 with a message naming the setting rather than leaving it to the OOM killer. Verified end to end — `--size small --train-modalities` completes all 200 image generator steps.
